x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/nodes/execute_tools.ts

```diff
@@ -38,7 +38,16 @@ export const executeTools = async ({ config, logger, state, tools }: ExecuteTool
   if (!agentAction || 'returnValues' in agentAction) {
     throw new Error('Agent has not been run yet');
   }
-  const out = await toolExecutor.invoke(agentAction, config);
+
+  let out;
+  try {
+    out = await toolExecutor.invoke(agentAction, config);
```

Comment:
to catch `Received tool input did not match expected schema` errors
<img width="1485" alt="image" src="https://github.com/elastic/kibana/assets/5188868/8d48bb16-6e17-4135-a26f-935e58ec13cb">